{"summary":{"outcome":"Failed","testsRan":3,"passing":1,"failing":2,"skipped":0,"passRate":"33%","failRate":"67%","skipRate":"0%","testStartTime":"2025-08-05T19:59:03.000Z","testSetupTimeInMs":1877,"testExecutionTimeInMs":1000,"testTotalTimeInMs":2877,"commandTimeInMs":1024,"hostname":"https://orgfarm-d89e3e6fd7-dev-ed.develop.my.salesforce.com","orgId":"00DgL00000713kbUAA","username":"ajayp9402351@agentforce.com","testRunId":"707gL00000B3qYY","userId":"005gL000004tq0UQAQ"},"tests":[{"id":"07MgL0000018mV8UAI","queueItemId":"709gL0000039iBNQAY","stackTrace":"Class.LeadCounterTest.TestLeadsCountWhenNotConverted: line 82, column 1","message":"System.SObjectException: SObject row was retrieved via SOQL without querying the requested field: Contact.LIC_Agent__c","asyncApexJobId":"707gL00000B3qYYQAZ","methodName":"TestLeadsCountWhenNotConverted","outcome":"Fail","apexLogId":null,"apexClass":{"id":"01pgL0000047xVhQAI","name":"LeadCounterTest","namespacePrefix":null,"fullName":"LeadCounterTest"},"runTime":983,"testTimestamp":"2025-08-05T19:59:05.000+0000","fullName":"LeadCounterTest.TestLeadsCountWhenNotConverted","diagnostic":{"exceptionMessage":"System.SObjectException: SObject row was retrieved via SOQL without querying the requested field: Contact.LIC_Agent__c","exceptionStackTrace":"Class.LeadCounterTest.TestLeadsCountWhenNotConverted: line 82, column 1","className":"LeadCounterTest","compileProblem":"","lineNumber":82,"columnNumber":1}},{"id":"07MgL0000018mV9UAI","queueItemId":"709gL0000039iBNQAY","stackTrace":"Class.LeadCounterTest.testLeadsCountWhenConverted: line 45, column 1","message":"System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, Owner ID: id value of incorrect type: 003gL000008PVYhQAO: [OwnerId]","asyncApexJobId":"707gL00000B3qYYQAZ","methodName":"testLeadsCountWhenConverted","outcome":"Fail","apexLogId":null,"apexClass":{"id":"01pgL0000047xVhQAI","name":"LeadCounterTest","namespacePrefix":null,"fullName":"LeadCounterTest"},"runTime":17,"testTimestamp":"2025-08-05T19:59:06.000+0000","fullName":"LeadCounterTest.testLeadsCountWhenConverted","diagnostic":{"exceptionMessage":"System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, Owner ID: id value of incorrect type: 003gL000008PVYhQAO: [OwnerId]","exceptionStackTrace":"Class.LeadCounterTest.testLeadsCountWhenConverted: line 45, column 1","className":"LeadCounterTest","compileProblem":"","lineNumber":45,"columnNumber":1}}]}